web/docs/database/entities.md

```diff
@@ -1,3 +1,66 @@
 ---
 title: Entities
 ---
+
+Entities are the foundation of your app's data model. In short, an Entity defines a model in your database.
+
+Wasp uses the excellent [Prisma ORM](https://www.prisma.io/) to implement all database functionality and occasionally enhances it with a thin abstraction layer.
+Wasp Entities directly correspond to [Prisma's data model](https://www.prisma.io/docs/concepts/components/prisma-schema/data-model). Still, you don't need to be familiar with Prisma to effectively use Wasp, as it comes with a simple API wrapper for working with Prisma's core features.
+
+The only requirement for defining Wasp Entities is familiarity with the **_Prisma Schema Language (PSL)_**, a simple definition language explicitly created for defining models in Prisma.
+The language is declarative and very intuitive. We'll also go through an example later in the text, so there's no need to go and thoroughly learn it right away. Still, if you're curious, look no further than Prisma's official documentation:
+
+- [Basic intro and examples](https://www.prisma.io/docs/reference/tools-and-interfaces/prisma-schema)
+- [A more exhaustive language specification](https://www.prisma.io/docs/reference/api-reference/prisma-schema-reference)
+
+## Defining an Entity
+
+As mentioned, an `entity` declaration represents a database model.
+
+Each `Entity` declaration corresponds 1-to-1 to [Prisma's data model](https://www.prisma.io/docs/concepts/components/prisma-schema/data-model). Here's how you could define an entity that represents a Task:
+
+```wasp
+entity Task {=psl
+    id          Int     @id @default(autoincrement())
+    description String
+    isDone      Boolean @default(false)
+psl=}
+```
+
+Let's go through this declaration in detail:
+
+- `entity Task` - This tells Wasp that we wish to define an Entity (i.e., database model) called `Task`. Wasp automatically creates a table called `tasks`.
+- `{=psl ... psl=}` - Wasp treats everything that comes between the two `psl` tags as [PSL](https://www.prisma.io/docs/reference/tools-and-interfaces/prisma-schema).
```

Comment:
```suggestion
- `{=psl ... psl=}` - Wasp treats everything that comes between the two `psl` tags as valid [Prisma schema language](https://www.prisma.io/docs/reference/tools-and-interfaces/prisma-schema).
```

Maybe expand from `PSL` to full name to keep in mind that this _is_ Prisma under the hood